src/main/java/com/hbm/items/special/ItemGlitch.java

```diff
@@ -157,82 +158,82 @@ public ItemStack onItemRightClick(ItemStack stack, World world, EntityPlayer pla
 				mirv.posY = player.posY + 100;
 				mirv.posZ = player.posZ;
 				world.spawnEntityInWorld(mirv);
-				player.addChatMessage(new ChatComponentText("Watch your head!"));
+				player.addChatMessage(new ChatComponentText(I18nUtil.resolveKey("item.glitch.mess.16")));
 				break;
 			case 29:
 				ExplosionLarge.spawnBurst(world, player.posX, player.posY, player.posZ, 27, 3);
-				player.addChatMessage(new ChatComponentText("Bam!"));
+				player.addChatMessage(new ChatComponentText(I18nUtil.resolveKey("item.glitch.mess.17")));
 				break;
 			case 30:
 				player.inventory.addItemStackToInventory(new ItemStack(ModItems.plate_saturnite));
-				player.addChatMessage(new ChatComponentText("It's dangerous to go alone, take this!"));
+				player.addChatMessage(new ChatComponentText(I18nUtil.resolveKey("item.glitch.mess.18")));
 				break;
 			}
-		
+
 		player.inventoryContainer.detectAndSendChanges();
-		
+
 		return stack;
 	}
-	
+
 	@Override
 	public void addInformation(ItemStack itemstack, EntityPlayer player, List list, boolean bool)
 	{
-		list.add("It's a gamble!");
+		list.add(I18nUtil.resolveKey("item.glitch.desc"));
 		list.add("");
 		switch(MainRegistry.polaroidID) {
-		case 1: 
-			list.add("Click-click-click!");
+		case 1:
+			list.add(I18nUtil.resolveKey("item.glitch.desc.1"));
```

Comment:
...at that point just do `"item.glitch.desc." + MainRegistry.polaroidID` instead of having the switch there